calico_containers/pycalico/tests/test_util.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from mock import patch, Mock, call, ANY
+from calico_ctl.node import _attach_and_stream
+from docker import Client
+import sys
+import signal
+import os
+from pycalico.util import get_host_ips
+
+mock_ip_addr = \
```

Comment:
Its typical to make global definitions like this in all caps - makes it easier to spot.